include/screen.h

```diff
@@ -1,589 +1,661 @@
+            uint8_t b = gamma5[color & 0x1F];
+
+            return lv_color_make(r, g, b);
+        }
+
+    public:
+
+        AMOLEDScreen(int w, int h) : Screen(w, h)
+        {
+            if (!amoled.begin())
+            {
+                debugE("AMOLED begin failed");
+                return;
+            }
+
+            amoled.setBrightness(255);
+            
+            // Register lvgl helper
+            beginLvglHelper(amoled);
+
+            const size_t kBufferSize = LV_CANVAS_BUF_SIZE_TRUE_COLOR(w,h);
+            cbuf = (lv_color_t *)heap_caps_malloc(kBufferSize, MALLOC_CAP_SPIRAM);
+            if (cbuf == NULL)
+            {
+                debugE("AMOLED malloc failed");
+                return;
+            }
+            debugW("Allocated %d bytes for lvgl canvas: %p\n", sizeof(lv_color_t) * w * h, cbuf);
+            
+            canvas = lv_canvas_create(lv_scr_act());
+            lv_canvas_set_buffer(canvas, cbuf, w, h, LV_IMG_CF_TRUE_COLOR);
+            lv_obj_center(canvas);
+            lv_canvas_fill_bg(canvas, lv_from16Bit(GREEN16), LV_OPA_COVER);            
+        }
+
+        ~AMOLEDScreen()
+        {
+            if (canvas)
+            {
+                lv_obj_del(canvas);
+                canvas = NULL;
+            }
+            if (cbuf)
+            {
+                heap_caps_free(cbuf);
+                cbuf = NULL;
+            }
+        }   
+
+        virtual void drawPixel(int16_t x, int16_t y, uint16_t color) override
+        {
+            assert(canvas != NULL);
+            assert(cbuf != NULL);
+
+            lv_color_t lv_color = lv_from16Bit(color); 
+            lv_canvas_set_px_color(canvas, x, y, lv_color);
+
+        }
+
+        virtual void fillRect(int16_t x, int16_t y, int16_t w, int16_t h, uint16_t color) override
+        {
+            assert(canvas != NULL);
+            assert(cbuf != NULL);
+
+            // Define the rectangle's properties
+            lv_draw_rect_dsc_t rect_dsc;
+            lv_draw_rect_dsc_init(&rect_dsc);
+            rect_dsc.bg_opa = LV_OPA_COVER;
+            rect_dsc.bg_color = lv_from16Bit(color); 
+
+            // Draw the rectangle
+            lv_canvas_draw_rect(canvas, x, y, w, h, &rect_dsc);
+        }
+
+        virtual void fillScreen(uint16_t color) override
+        {
+            assert(canvas);
+            assert(cbuf);
+
+            lv_color_t lv_color = lv_from16Bit(color); 
+            lv_canvas_fill_bg(canvas, lv_color, LV_OPA_COVER);
+        }
+
+    // AMOLED is a full color panel but we want a different default theme
+    virtual bool IsMonochrome() const override { return false; }
+    virtual uint16_t GetTextColor() const override { return Screen::to16bit(CRGB(100, 255, 20)); }
+    virtual uint16_t GetBkgndColor() const override { return Screen::to16bit(CRGB::Black); }
+    virtual uint16_t GetBorderColor() const override { return Screen::to16bit(CRGB::Red); }

```

Comment:
`virtual` is implied by `override`. Also whitespacing.

```suggestion
        // AMOLED is a full color panel but we want a different default theme
        bool IsMonochrome() const override
        {
            return false;
        }

        uint16_t GetTextColor() const override
        {
            return Screen::to16bit(CRGB(100, 255, 20));
        }

        uint16_t GetBkgndColor() const override
        {
            return Screen::to16bit(CRGB::Black);
        }

        uint16_t GetBorderColor() const override
        {
            return Screen::to16bit(CRGB::Red);
        }
```